modules/promotion/src/Plugin/Commerce/PromotionCondition/PromotionConditionInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Drupal\commerce_promotion\Plugin\Commerce\PromotionCondition;
+
+use Drupal\Core\Condition\ConditionInterface;
+
+/**
+ * Defines an interface for Condition plugins.
+ *
+ * Extends the core Condition interface to support Commerce Promotion specific
+ * requirements.
+ */
+interface PromotionConditionInterface extends ConditionInterface {
+
+  /**
+   * Returns what entity the promotion offer applies to.
```

Comment:
We use "Gets" instead of "Returns". And this returns the entity type, not the entity. Perhaps appliesTo() is not descriptive enough? getTargetEntityType() would not require me to read the docblock.